postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -484,3 +526,35 @@ def _explain_pg_stat_activity(self, rows):
                     1,
                     tags=self._tags + ["error:collect-plan-for-statement-crash"],
                 )
+
+    def _load_query_max_text_size(self, db):
+        try:
+            with db.cursor(cursor_factory=psycopg2.extras.DictCursor) as cursor:
+                self._log.debug("Running query [%s]", MAX_ACTIVITY_QUERY_SIZE_QUERY)
+                cursor.execute(MAX_ACTIVITY_QUERY_SIZE_QUERY)
+                row = cursor.fetchone()
+                self._max_query_size = int(row['setting'])
+        except (psycopg2.DatabaseError, psycopg2.OperationalError) as e:
+            self._log.warning("cannot read track_activity_query_size from pg_settings: %s", repr(e))
+            self._check.count(
+                "dd.postgres.statement_samples.error",
+                1,
+                tags=self._tags + ["error:load-track-activity-query-size"],
+            )
+
+    @staticmethod
+    def _is_statement_truncated(max_query_size, statement):
```

Comment:
I don't like this function name because it doesn't return a boolean but a StatementTruncationState.
Can we rename it to `get_trunctation_state`?

Reply:
Thanks for the suggestion. Update ✅